Jenkinsfile

```
@@ -197,9 +199,14 @@ def postRelease(){
     sh(label: 'Set branch', script: """#!/bin/bash
       git checkout -b ${BRANCH_NAME}
     """)
-    gitCreateTag(tag: "${env.GO_VERSION}", pushArgs: '--force')
-    withCredentials([string(credentialsId: '2a9602aa-ab9f-4e52-baf3-b71ca88469c7', variable: 'GREN_GITHUB_TOKEN')]) {
-      sh(label: 'Creating Release Notes', script: '.ci/scripts/release-notes.sh')
+    try {
+      gitCreateTag(tag: "${env.GO_VERSION}", pushArgs: '--force')
+      withCredentials([string(credentialsId: '2a9602aa-ab9f-4e52-baf3-b71ca88469c7', variable: 'GREN_GITHUB_TOKEN')]) {
+        sh(label: 'Creating Release Notes', script: '.ci/scripts/release-notes.sh')
+      }
+    } catch (e) {
+      // Probably the tag already exists
+      log(level: 'INFO', text: "postRelease failed with message : ${e?.message}")
```

Is `INFO` the right level for a failure message? Perhaps change the log level?

---

I'll change it to `WARN`, theoretically this should not happen since `isNewRelease()` should be the one driving the release, but somehow it returned empty values in the very first release.

```suggestion
      log(level: 'WARN', text: "postRelease failed with message : ${e?.message}")
```